velox/common/base/RawVector.h

```diff
@@ -22,13 +22,27 @@
 
 namespace facebook::velox {
 
+template <typename T, uint8_t kAlignment>
+class StdAlignedAllocator {
+ public:
+  T* allocate(size_t n) {
+    return reinterpret_cast<T*>(aligned_alloc(kAlignment, n));
+  }
+
+  void deallocate(T* data, /*unused */ size_t n) {
+    ::free(data);
+  }
+};
+
 /// Class template similar to std::vector with no default construction and a
 /// SIMD load worth of padding below and above the data. The idea is that one
 /// can access the data at full SIMD width at both ends.
-template <typename T>
+template <
+    typename T,
+    typename Allocator = StdAlignedAllocator<T, simd::kPadding>>
```

Comment:
Let's remove the copy constructor and assignment and fix the move semantics to move the allocator as well